doc/tutorials/external_memory.rst

```diff
@@ -1,6 +1,6 @@
-############################################
-Using XGBoost External Memory Version (beta)
```

Comment:
Em ... the external memory support is not available for CPU Hist, if we were to declare it stable, we should at least document it in the first section that `Hist` is not supported.

Reply:
Are you sure? I think I've tried `hist` for comparison in external memory mode. The limitation is right now the CPU code keeps the whole histogram in memory, which may still cause out-of-memory errors if the dataset is too large, but the mechanism of using `#cacheprefix` and dealing with multiple pages is supported.

Reply:
I'm not.  But there's https://github.com/dmlc/xgboost/issues/4093

Reply:
Ok, added `hist` not well tested as a limitation. If you want, I can add the `beta` back. :)